src/node_worker.h

```diff
@@ -89,10 +99,15 @@ class Worker : public AsyncWrap {
   mutable Mutex mutex_;
 
   bool thread_joined_ = true;
+  const char* custom_error_ = nullptr;
   int exit_code_ = 0;
   uint64_t thread_id_ = -1;
   uintptr_t stack_base_;
 
+  // Custom resource constraints:
+  double resource_limits_[kTotalResourceLimitCount];
```

Comment:
Is there a reason not to use `AliasedBuffer` here?

Reply:
I mean… is there any reason to do that? We don’t need to access these values after we’ve stored them, so an AliasedBuffer seems like unnecessary complexity?